src/platform-includes/distributed-tracing/custom-instrumentation/android.mdx

```diff
@@ -0,0 +1,64 @@
+On this page you will learn how to manually propagate trace information into and out of your Android application. Please note, that you do not need to do this manually, if you [use one of our supported frameworks](/platforms/android/usage/distributed-tracing/#how-to-use-distributed-tracing), or you have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.
+
+To set it up manually, all you have to do is to make sure your application extracts incoming tracing information and to attach it again when making an outgoing request within your application.
+
+Your Android application is likely going to be the first in a chain of requests which means there probably won't be any tracing information to extract but instead your application starts a new trace and sends out tracing information so other applications in the chain can keep the trace going.
+
+## Step 1) Extract Incoming Tracing Information
```

Comment:
Not sure if we even want to display this section for Android as it'll probably be a very niche use case to extract tracing information on Android.

Reply:
I also added a paragraph above explaining this.

Reply:
@markushi @romtsn may I have your opinion on ^

Reply:
Sounds good to me!